### PR TITLE
[FIX] Fix `no-return-assign` linting error

### PR DESCRIPTION
**Pull request by davidlamt**
## Description

Refactor `memoMarked()` to resolve `no-return-assign` eslint error and hopefully improve readability.

## What does your PR belong to?

* [ ] Questions / Answers
* [x] Website
* [ ] General / Things regarding the repository (like CI Integration)

## Types of changes

* [x] Bug fix (non-breaking change which fixes an issue)
* [ ] Enhancement (non-breaking improvement of a question)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

* [x] My code follows the code style of this project.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [x] I have checked that the changes are working properly
* [x] I have checked that there isn't any PR doing the same
* [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: website/js/components/Markdown.js [hunk 1]
```diff
@@ -12,8 +12,9 @@ const setInnerHTML = html => el => {
 }
 
 const memoMarked = content => {
-  const result = markStore[content]
-  return result || (markStore[content] = marked(content))
+  if (!markStore[content]) markStore[content] = marked(content)
```
Comment (skatcat31) [inline website/js/components/Markdown.js]: Can we change the check to an `Object#hasOwnProperty()` call instead? This is a more robust test that will not fail if the property evaluates to falsey

```suggestion
  if (!markStore.hasOwnProperty(content)) markStore[content] = marked(content) 
```

Comment (atomiks) [reply website/js/components/Markdown.js]: Well the markdown content is never empty so doesn't really matter

Comment (skatcat31) [reply website/js/components/Markdown.js]: Fair enough, and there is no other way to put that key in there at the momen

**Comment by atomiks** [comment]
`Object.create(null)` creates an object without a prototype (so no `hasOwnProperty` method)

Change to this:
```js
const memoMarked = content => {
  if (content in markStore) {
    return markStore[content]
  } else {
    const result = marked(content)
    markStore[content] = result
    return result
  }
}
```

**Comment by davidlamt** [comment]
@fejes713,

Sorry, completely my fault that I did not re-test after accepting the suggested changes.

Feel free to update your PR #129 to use my original change:

```js
const memoMarked = content => {
  if (!markStore[content]) markStore[content] = marked(content)

  return markStore[content]
}
```

or @atomiks suggestion above. I have tested both variations - the ESLint error is removed and the build is successful.

Again, sorry for inconvenience!

**Pull request merged**